src/staff/hooks/useStaffMutation.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  useStaffMemberDeleteMutation,
+  useStaffMemberUpdateMutation,
+} from "@dashboard/graphql";
+import useNavigator from "@dashboard/hooks/useNavigator";
+import useNotifier from "@dashboard/hooks/useNotifier";
+import { commonMessages } from "@dashboard/intl";
+import { useIntl } from "react-intl";
+
+import { staffListUrl } from "../urls";
+
+export const useStaffMutation = () => {
```

Comment:
**issue**: naming convention - you call it mutation while you are calling two mutations, perhaps useStaffOperations would be cleaner? or something else?